### PR TITLE
feat: exporting gor-data-formatter to gorila-ui-components

### PR DESCRIPTION
**Pull request by tigredonorte**

### CODE REVIEW

**Comment by guilhermejcgois** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/guilhermejcgois/gorila-ui-components/pullRequest?prid=4491311&bid=15147276)

**Comment by guilhermejcgois** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/guilhermejcgois/gorila-ui-components/pullRequest?prid=4491311&bid=15147276)

File: README.md [hunk 1]
```diff
@@ -12,3 +12,27 @@ This is the home for the Gorila team's components, built by developers for devel
 See our [Getting Started Guide](./additional-documentation/getting-started.html) if you're building a project with ours widgets.
 
 If you'd like to contribute, you must follow our [contributing guidelines](./additional-documentation/contributing.html).
+
+### Developing
```
Comment (guilhermejcgois) [inline README.md]: These documentation needs to be in "Development Documentation" in http://ui.gorilainvest.com.br/docs, can you move it please?

Comment (guilhermejcgois) [reply README.md]: Also, the refered steps must be done by ng/nx cli, and not manually

File: CODEOWNERS [hunk 2]
```diff
@@ -5,6 +5,7 @@ libs/ui/button @guilhermejcgois @oliveira-gust
 libs/ui/card-container @tigredonorte
 libs/ui/countdown-button @guilhermejcgois @oliveira-gust
 libs/ui/enhanced-title @tigredonorte
+libs/ui/gor-data-formatter @tigredonorte
```
Comment (guilhermejcgois) [inline CODEOWNERS]: Note that all libs doesn't have prefix in folder and filename, mantain the pattern please

File: debug.log [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[1113/095750.014:ERROR:process_reader_win.cc(151)] SuspendThread: Acesso negado. (0x5)
```
Comment (guilhermejcgois) [inline debug.log]: Remove file and add it to ;gitignore. Which tool are generating this debug fiile?

File: libs/shell/presentational/src/lib/presentational-shell.module.ts [hunk 4]
```diff
@@ -1,44 +1,92 @@
...
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatListModule } from '@angular/material/list';
+import { MatSelectModule } from '@angular/material/select';
+import { RouterModule } from '@angular/router';
+import { GdkModule } from '@gorilainvest/gdk';
+
```
Comment (guilhermejcgois) [inline libs/shell/presentational/src/lib/presentational-shell.module.ts]: Remove all these empty lines

File: libs/shell/presentational/src/lib/ui/gor-data-formatter/gor-data-formatter.component.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+export class GorDataFormatterComponent implements OnInit {
+  dataTypeFormControl = new FormControl('date');
+  valueFormControl = new FormControl('2019-01-02');
+  dataOptionFormControl = new FormControl('1.2-2');
+  dataStyleFormControl = new FormControl('');
+  displayDataOption = false;
+  isCustom = false;
+
+  constructor() {
+    this.dataTypeFormControl.valueChanges.subscribe(data => this.checkType(data));
```
Comment (guilhermejcgois) [inline libs/shell/presentational/src/lib/ui/gor-data-formatter/gor-data-formatter.component.ts]: Is a bad practice to do subscribes on class constructors, do it in another place instead

**Comment by guilhermejcgois** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/gorila-ui-components/gorila-ui-components/pullRequest?prid=4491311&bid=15147276)

File: README.md (hunk 1 above)
Comment (tigredonorte) [reply README.md]: I think that is very wired the readme file doesn't contains the information about how to use a repository.

File: debug.log (hunk 3 above)
Comment (tigredonorte) [reply debug.log]: I don't know.

File: README.md (hunk 1 above)
Comment (guilhermejcgois) [reply README.md]: Okay, but this is out of scope for this pull request, and wasn't made with a specific commit, with this I mean that this file was mixed with another unrelated files in the same commit.

Comment (guilhermejcgois) [reply README.md]: if this were the pattern within this repository, contribution guidelines and get started should already be in the readme, agree?

Comment (tigredonorte) [reply README.md]: > Also, the refered steps must be done by ng/nx cli, and not manually

Where's the instructions? I can't find this steps in documentation. Also, I can't find documentation of existent components.

File: CODEOWNERS (hunk 2 above)
Comment (tigredonorte) [reply CODEOWNERS]: ok

File: libs/shell/presentational/src/lib/presentational-shell.module.ts (hunk 4 above)
Comment (tigredonorte) [reply libs/shell/presentational/src/lib/presentational-shell.module.ts]: ok

File: libs/shell/presentational/src/lib/ui/gor-data-formatter/gor-data-formatter.component.ts (hunk 5 above)
Comment (tigredonorte) [reply libs/shell/presentational/src/lib/ui/gor-data-formatter/gor-data-formatter.component.ts]: ok

File: README.md (hunk 1 above)
Comment (guilhermejcgois) [reply README.md]: http://ui.gorilainvest.com.br/docs/

Comment (tigredonorte) [reply README.md]: In this link you didn't explain the two indirect questions above:
1) where is the information about how to use ng/nx cli to create a new component without do steps manually?
2) how to add information in the docs?

**Pull request merged**